### PR TITLE
Paramiko to replace standard Python FTP library

### PR DESCRIPTION
**Pull request by Dziolas**

### CODE REVIEW

**Comment by jalavik** [comment]
Perhaps change the commit message of `elsevier: some changeI don't remeber what was doing` or squash it into the ftp_utils commit as it is only removing whitespace basically.

File: harvestingkit/ftp_utils.py [hunk 1]
```diff
@@ -118,14 +124,15 @@ def download(self, source_file, target_folder=''):
         source_file = os.path.basename(source_file)
         destination = join(destination_folder, source_file)
         try:
-            with open(destination, 'wb') as result:
-                self._ftp.retrbinary('RETR %s' % (source_file,),
-                                     result.write)
+            # with open(destination, 'wb') as result:
```
Comment (jalavik) [inline harvestingkit/ftp_utils.py]: Guess these comments can be removed.

File: harvestingkit/ftp_utils.py [hunk 2]
```diff
@@ -260,19 +268,14 @@ def get_filesize(self, filename):
...
-        self._ftp.dir(filename, dir_callback)
-        return result[0]
+        return self._sftp_client.lstat(filename).st_size
 
     def get_datestamp(self, filename):
-        datestamp = self._ftp.sendcmd('MDTM ' + filename)
-        datestamp = datetime.strptime(datestamp[4:],
-                                      "%Y%m%d%H%M%S").strftime("%Y-%M-%d")
-        return datestamp
+        # datestamp = self._ftp.sendcmd('MDTM ' + filename)
```
Comment (jalavik) [inline harvestingkit/ftp_utils.py]: Guess these comments can be removed.

File: harvestingkit/ftp_utils.py [hunk 3]
```diff
@@ -44,25 +45,29 @@ class FtpHandler(object):
                                for authentication with the server.
     :type netrc_file: string
     """
-    def __init__(self, server, username='', passwd='', netrc_file=''):
+    def __init__(self, server, username='', passwd='', netrc_file='', sftp=False):
+        if sftp:
```
Comment (jalavik) [inline harvestingkit/ftp_utils.py]: shall we instead pass a parameter `port` instead of boolean `sftp`?

**Comment by Dziolas** [comment]
Do not merge it I made a terrible and stupid mistake. I will make a new version soon.

**Pull request closed without merging**